package.json

```diff
@@ -28,31 +28,34 @@
   "devDependencies": {
     "asar": "^0.9.0",
     "babel-core": "^6.4.5",
-    "babel-eslint": "^4.1.6",
+    "babel-eslint": "^4.1.8",
     "babel-loader": "^6.2.2",
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "chai": "^3.4.1",
-    "electron-builder": "^2.6.0",
+    "electron-builder": "^2.8.4",
     "electron-mocha": "^0.8.0",
-    "electron-packager": "git://github.com/mawie81/electron-packager#mas-support",
-    "electron-prebuilt": "^0.36.7",
-    "eslint": "^1.7.2",
-    "eslint-plugin-react": "^3.14.0",
+    "electron-packager": "^5.2.1",
+    "electron-prebuilt": "0.36.8",
+    "eslint": "^1.10.3",
+    "eslint-plugin-react": "^3.16.1",
     "json-loader": "^0.5.4",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
     "mv": "^2.1.1",
     "unzip": "^0.1.11",
-    "webpack": "^1.12.12"
+    "webpack": "^1.12.14"
   },
   "dependencies": {
+    "babel-core": "^6.6.0",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.6.0",
     "debug": "^2.2.0",
     "express": "^4.13.3",
     "jade": "^1.11.0",
     "lodash.debounce": "^4.0.0",
     "portscanner": "^1.0.0",
-    "spellchecker": "^3.1.3",
-    "superagent": "^1.4.0"
+    "spellchecker": "^3.2.3",
+    "superagent": "^1.7.2"
```

Comment:
There's a lot of dependencies changing here which seems like it's not intentional.